packages/slate-react/src/plugin/with-react.ts

```diff
@@ -56,6 +57,91 @@ export const withReact = <T extends Editor>(editor: T) => {
     }
   }
 
+  e.setFragmentData = (data: DataTransfer) => {
```

Comment:
This method is an exact copy of `setFragmentData` with `editor` replaced by `e`, and one other change noted below: